pkgs/development/python-modules/pykerberos/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchPypi, buildPythonPackage, krb5 }:
+
+buildPythonPackage rec {
+    pname = "pykerberos";
+    version = "1.2.1";
+    name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.